test/resources/case11_policyset_controller/case11-statuscheck-8.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: policy.open-cluster-management.io/v1beta1
+kind: PolicySet
+metadata:
+  name: case11-test-policyset
+spec:
+  policies:
+  - case11-test-policy
+status:
+  compliant: Pending
+  placement:
+  - placementBinding: case11-test-policyset-pb
+    placementRule: case11-test-policyset-plr
+  statusMessage: All policies are reporting status
```

Comment:
Optional: This might be a bit misleading since there are policies that are pending.

Reply:
I wasn't sure whether to change this since pending is technically reporting status. Do you think it would be better to list the pending policies in the status message like we do for disabled policies? @mprahl @JustinKuli